src/nupic/bindings/algorithms.i

```diff
@@ -101,9 +101,20 @@ _ALGORITHMS = _algorithms
 #include <nupic/proto/ConnectionsProto.capnp.h>
 #include <nupic/proto/SpatialPoolerProto.capnp.h>
 #include <nupic/proto/TemporalMemoryProto.capnp.h>
+%}
 
+//
+// Numpy API
+//
+%{
 #define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION
 #include <numpy/arrayobject.h>
+%}
+%init %{
+  import_array();
```

Comment:
@mrcslws, Interesting, numpy used to work without having this `import_array` call here. I wonder what changed. Do you understand why it became necessary? There are existing calls to `import_array` and `_import_array` elsewhere in the source code. Are they necessary after this change?